src/components/Section/Foreign/Activities/Support.jsx

```diff
@@ -42,7 +42,8 @@ export default class Support extends SubsectionElement {
   }
 
   summary(item, index) {
-    const name = NameSummary((item || {}).Name)
+    const o = (item || {}).Item || {}
```

Comment:
The final ` || {}` on this is not needed since it'll already get set to an empty object inside the parentheses.